internal/peer/lifecycle/chaincode/install.go

```diff
@@ -55,7 +55,7 @@ func (i *InstallInput) Validate() error {
 // InstallCmd returns the cobra command for chaincode install.
 func InstallCmd(i *Installer, cryptoProvider bccsp.BCCSP) *cobra.Command {
 	chaincodeInstallCmd := &cobra.Command{
-		Use:       "install",
+		Use:       "install [packageFiles]",
```

Comment:
```suggestion
		Use:       "install [packageFile]",
```

Reply:
I have corrected my changes and update with a new commit.

thanks again for your help